tools/build-osbs/build-osbs.sh

```diff
@@ -259,6 +259,6 @@ if [ -n "$OSBS_BUILD_USER" ]; then
 fi
 
 # Invoke cekit and respond with Y to any prompts
-echo cekit $debug --overrides-file branch-overrides.yaml $overrides $artifactoverrides --build-engine=osbs --build-osbs-target=$OSBS_BUILD_TARGET $builduser --work-dir "$cekit_cache_dir" $CEKIT_BUILD_OPTIONS build
+echo cekit build $debug --overrides-file branch-overrides.yaml $overrides $artifactoverrides --build-engine=osbs --build-osbs-target=$OSBS_BUILD_TARGET $builduser --work-dir "$cekit_cache_dir" $CEKIT_BUILD_OPTIONS docker
 
-yes Y | cekit $debug --overrides-file branch-overrides.yaml $overrides $artifactoverrides --build-engine=osbs --build-osbs-target=$OSBS_BUILD_TARGET $builduser --work-dir "$cekit_cache_dir" $CEKIT_BUILD_OPTIONS build
+yes Y | cekit build $debug --overrides-file branch-overrides.yaml $overrides $artifactoverrides --build-engine=osbs --build-osbs-target=$OSBS_BUILD_TARGET $builduser --work-dir "$cekit_cache_dir" $CEKIT_BUILD_OPTIONS docker
```

Comment:
The command is wrong...
CEKIT_BUILD_OPTIONS does n ot apply anymore.

$overrides $artifactoverrides: needs to be tested since it has changed on cekit3
--build-engine=osbs: that not correct anymore
docker build engine should not be used if you intended to use osbs.
--build-osbs-target is not koji-target

should be something like
cekit $verbose --work-dir="$cekit_cache_dir" **build** --overrides-file branch-overrides.yaml **osbs** --user=$builduser --build-osbs-target $OSBS_BUILD_TARGET